diff -up python-openapi-core-0.22.0/pyproject.toml.omv~ python-openapi-core-0.22.0/pyproject.toml
--- python-openapi-core-0.22.0/pyproject.toml.omv~	2026-03-01 14:56:10.876684098 +0100
+++ python-openapi-core-0.22.0/pyproject.toml	2026-03-01 14:56:18.151196023 +0100
@@ -68,7 +68,7 @@ aiohttp = {version = ">=3.0", optional =
 starlette = {version = ">=0.26.1,<0.50.0", optional = true}
 isodate = "*"
 more-itertools = "*"
-openapi-schema-validator = "^0.6.0"
+openapi-schema-validator = "^0.8.0"
 openapi-spec-validator = {version = "^0.8.0b3", allow-prereleases = true}
 requests = {version = "*", optional = true}
 werkzeug = ">=2.1.0"
